src/workloads/execution/LookupSBEPushdown.yml

```diff
@@ -0,0 +1,408 @@
+  LoadConfig:
+    Path: "../../phases/execution/LookupCommands.yml"
+    Key: NLJLookupCmdTemplate
+    Parameters:
+      l: *LocalSmallDoc100k
+      f: *Foreign5500
+
+NLJLookupForeign7500Cmd: &NLJLookupForeign7500Cmd
+  LoadConfig:
+    Path: "../../phases/execution/LookupCommands.yml"
+    Key: NLJLookupCmdTemplate
+    Parameters:
+      l: *LocalSmallDoc100k
+      f: *Foreign7500
+
+NLJLookupForeign9500Cmd: &NLJLookupForeign9500Cmd
+  LoadConfig:
+    Path: "../../phases/execution/LookupCommands.yml"
+    Key: NLJLookupCmdTemplate
+    Parameters:
+      l: *LocalSmallDoc100k
+      f: *Foreign9500
+
+# Defines test operations for 'HJLookup' actor.
+HJTestOperations: &HJTestOperations
+- OperationMetricsName: Foreign1500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *HJLookupForeign1500Cmd
+- OperationMetricsName: Foreign3500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *HJLookupForeign3500Cmd
+- OperationMetricsName: Foreign5500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *HJLookupForeign5500Cmd
+- OperationMetricsName: Foreign7500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *HJLookupForeign7500Cmd
+- OperationMetricsName: Foreign9500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *HJLookupForeign9500Cmd
+
+# Defines test operations for 'NLJLookup'/ 'INLJLookup' actors.
+NLJTestOperations: &NLJTestOperations
+- OperationMetricsName: Foreign1500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign1500Cmd
+- OperationMetricsName: Foreign3500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign3500Cmd
+- OperationMetricsName: Foreign5500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign5500Cmd
+- OperationMetricsName: Foreign7500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign7500Cmd
+- OperationMetricsName: Foreign9500_IntTopField
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign9500Cmd
+
+# Defines how many times each test operation is executed.
+TestRepeatCount: &TestRepeatCount 10
+
+Actors:
+# The 'Setup' actor drops the database and runs in the 0th phase.
+- Name: Setup
+  Type: RunCommand
+  Database: *db
+  Thread: 1
+  Phases:
+  - Repeat: 1
+    Threads: 1
+    Database: *db
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand: {dropDatabase: 1}
+  - Phase: 1..5
+    Nop: true
+
+# The 'LoadForeign1500' / 'LoadForeign3500' / 'LoadForeign5500' / 'LoadForeign7500' /
+# 'LoadForeign9500' / 'LoadLocalSmallDoc100k' actors runs in the 1st phase.
+- Name: LoadForeign1500
+  Type: CrudActor
+  Database: *db
+  Threads: 1
+  Phases:
+  - &Nop {Nop: true}
+  - Repeat: 1500
+    Threads: 1
+    Collection: *Foreign1500
+    Operations:
+    - OperationName: insertOne
+      OperationCommand:
+        Document: *ForeignSmallDoc1500
+  - Phase: 2..5
+    Nop: true
+
+- Name: LoadForeign3500
```

Comment:
I'm confused as to why these are different actors. Is it just to parallelize?

Reply:
Because I need to create different collections with the different name, different cardinality, different document type. How can I achieve these scenarios using one actor?

Reply:
I see. That makes sense. It would be great to be able to specify a collection name for `Loader`, which was a prohibiting factor for me to use `Loader` for my scenarios. Let me keep it as is for now.